### DISCUSSION TITLE
Alternative for SQL

### DISCUSSION THREAD

**Discussion by amingomezd** [typegoose/mongodb-memory-server#703] [General]
I use this package in every mongo project that I work with, but there is another projects that use mysql, postgres, mariaDB, and I can't find a memory server like this. Do you know one?

**Comment by hasezoey** [comment]
I dont know of any, but i also am not really active in the SQL space (aside from SQLite, and that just barely), so the projects wanting to use that would probably need to manually download and start sql server instances, or use whatever the CI provides
